.github/CONTRIBUTING.md

```diff
@@ -274,7 +274,13 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 #### Use
 * Bitwise AND - '&'
 	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
-
+* Do not explicitly assign `null` to newly declared variables as this is already their default value
+* When defining new datums, place newly defined variables at the bottom
```

Comment:
I'd say this should have an exception to it.

There's some instances where having a var that is newly declared be with previously declared ones.

for instance, if there's an object that has a lot of different icon states, and uses something like `base_icon`, it's *far* more readable to have that variable with `icon` and `icon_state` than off by itself at the bottom.

Generally a good rule, but I don't think it should be adhered to blindly.